tests/integration/instrumentation/test_worker_instrumentation.py

```diff
@@ -0,0 +1,44 @@
+import time
+
+from jina import Flow
+from tests.integration.instrumentation import ExecutorTestWithTracing, get_traces
+
+
+def test_span_order(jaeger_port, otlp_collector, otlp_receiver_port):
+    # cannot use the pythonic f = Flow(...) because an actual gateway process is not started
```

Comment:
I do not understand this

Reply:
This might be a bug or something that is part of the orchestrate layer but if I start as flow using:
```python
f = Flow(tracing=True, traces_exporter_host='localhost', traces_exporter_port=4317)

with f:
  f.post(...)
```

This doesn't produce the proper chain (v.s. flow.yml) of requests to the executor.

Reply:
I tested this again on my local machine and I cannot reproduce this issue. Maybe something was broken in the test that caused me to use teh `flow.yml`.

Reply:
can u get back to the Pythonic way then?

Reply:
Yes, the code has already been replaced with the pythonic code.